spec/client_spec.rb

```diff
@@ -4,14 +4,7 @@
   let (:client) {Takeout::Client.new(uri: 'test.com', endpoints: {get: ENDPOINTS, post: ENDPOINTS, put: ENDPOINTS, delete: ENDPOINTS})}
 
   context 'initialization' do
-    it 'creates methods for all provided endpoints' do
-      expect((client.methods - Object.methods - Takeout::Client.instance_methods(false))).to eq([:get_posts, :get_fake_failure, :get_fake_missing, :get_fake_redirect,
-                                                                                                 :post_posts, :post_fake_failure, :post_fake_missing, :post_fake_redirect,
-                                                                                                 :put_posts, :put_fake_failure, :put_fake_missing, :put_fake_redirect,
-                                                                                                 :delete_posts, :delete_fake_failure, :delete_fake_missing, :delete_fake_redirect])
-    end
-
-    it 'yeilds when block is given' do
+    it 'yields when block is given' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.